docs_2.0/manual_EN/4.ngql-guide/15.subgraph-and-path/1.get-subgraph.md

```diff
@@ -0,0 +1,78 @@
+# GET SUBGRAPH
+
+The `GET SUBGRAPH` statement retrieves information of vertices and edges reachable from the start vertex over the specified edges.
+
+## Syntax
+
+```nGQL
+GET SUBGRAPH [<step_count> STEPS] FROM {<vid>, <vid>...}
+[IN <edge_type>, <edge_type>...]
+[OUT <edge_type>, <edge_type>...]
+[BOTH <edge_type>, <edge_type>...]
+```
+
+|Clause|Description|
+|-|-|
+|`STEPS`|Specifies the steps to go from the start vertices. A `step_count` must be a non-negative integer. Its default value is 1.|
+|`FROM`|Specifies the start vertices.|
+|`IN`|Gets the subgraphs from the start vertices over the specified ingoing edges (edges pointing to the start vertices).|
+|`OUT`|Gets the subgraphs from the start vertices over the specified outgoing edges (edges pointing out from the start vertices).|
+|`BOTH`|Gets the subgraphs from the start vertices over the specified types of edges, both ingoing and outgoing.|
+
+## Examples
+
+The following graph is used as the sample.
+
+![A sample graph for GET SUBGRAPH](https://docs-cdn.nebula-graph.com.cn/docs_2.0/3.quick-start/demo_dataset_for_quick_start.png)
+
+* Go one step from the vertex with VID "player100" over all types of edges and get the subgraph.
```

Comment:
maybe "player101" is a better example? It has edges of both types and both directions.